velox/functions/prestosql/window/AggregateWindow.cpp

```diff
@@ -153,12 +153,13 @@ class AggregateWindowFunction : public exec::WindowFunction {
     vector_size_t lastRow = rawFrameEnds[0];
 
     bool nonDecreasingFrameEnd = true;
+    bool fixedFirstRow = true;
```

Comment:
In a follow-up PR, consider passing BoundType to the WindowFunction so it can skip this loop altogether.

Reply:
Thanks for the suggestion, I was also thinking it would be easier to pass the window frame or the bound type and remove this loop, will make the change in a subsequent PR.